de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterWindowEventHandler.java

```diff
@@ -212,12 +213,17 @@ public void onRequestSimpleDownload(RequestSimpleDownloadEvent event) {
     public void onRequestSimpleUpload(RequestSimpleUploadEvent event) {
         Folder uploadDest = event.getDestinationFolder();
         if (canUpload(uploadDest)) {
-            new SimpleFileUploadDialog(uploadDest,
-                                       diskResourceServiceProvider.get(),
-                                       eventBus,
-                                       diskResourceUtil,
-                                       UriUtils.fromTrustedString(clientConstants.fileUploadServlet()),
-                                       userInfo.getUsername()).show();
+            SimpleFileUploadDialog dialog = new SimpleFileUploadDialog(uploadDest,
+                                                                        diskResourceServiceProvider.get(),
+                                                                        eventBus,
+                                                                        diskResourceUtil,
+                                                                        UriUtils.fromTrustedString(
+                                                                                clientConstants.fileUploadServlet()),
+                                                                        userInfo.getUsername());
+
+            dialog.setSize("500","400");
+            dialog.ensureDebugId(CommonsModule.UploadIds.BASE_ID);
```

Comment:
Also minor, but I've been following the pattern set by the Data, Apps, and Analysis windows where the dialog/window has control over setting its size and also sets its own id.